macros/utils/graph/get_package_database_and_schema.sql

```diff
@@ -1,11 +1,7 @@
 {% macro get_package_database_and_schema(package_name='elementary') %}
```

Comment:
The previous code supported all packages and the new code supports only Elementary. The new code returns only the Elementary db and schema (note that we can provide a different package name). 
I also don't really like relying on a specific model name in the package as this could break over time if we will remove / rename this model.

Reply:
I'll revert the changes.

The rational was that we use this macro a lot throughout the code,
I'm not sure if this is micro-optimization, but iterating the graph each time to find an Elementary model and extract its package and schema seems wasteful when you can do direct access, especially if the dbt project is big.